src/tls_wrap.cc

```diff
@@ -816,6 +816,49 @@ void TLSWrap::SetServername(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void TLSWrap::GetEphemeralKeyInfo(const FunctionCallbackInfo<Value>& args) {
```

Comment:
May I kindly suggest moving this to `SSLWrap` class in `node_crypto.cc`? This way it could be exposed for both `_tls_legacy.js` and `_tls_wrap.js`.